templates/CTCORECHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## CleverTap Android SDK CHANGE LOG
 
+### Version 4.5.1 (July 11, 2022)
+* New Feature : You can now call `onUserLogin`, `incrementValue` and `decrementValue` method via WebView Interface.
+* Improvement : Updated SSL Pinning Certificates
+
 ### Version 4.5.0 (June 3, 2022)
```

Comment:
Add a new changelog in 4.5.0 once the jacoco issue is solved in 4.5.1 @root-ansh-ct